### PR TITLE
Add config validation and fix bug

### PR DESCRIPTION
**Pull request by CMDRyan**
@Steven-Chan 

- App.js will now send a warning message if config exists an empty property
- Fixed "=== null" to "==null"

- Added .env-example file

- Updated README instructions

### CODE REVIEW

File: src/app.js [hunk 1]
```diff
@@ -11,6 +11,9 @@ const config = require("./config");
 const app = express();
 
 const { origin, AppID, APP_SECRET, redirect_url, cookieAge } = config;
+if([origin, AppID, APP_SECRET, redirect_url, cookieAge].includes("")){
+  console.log("[WARNING] config.js has at least one empty property.")
```
Comment (Steven-Chan) [inline src/app.js:15]: This is not log level warning. You need to use `console.warning`.

**Review by Steven-Chan — CHANGES_REQUESTED** [review]

**Comment by CMDRyan** [comment]
@Steven-Chan 

- The console.log message has been changed to a log level warning

File: src/app.js [hunk 2]
```diff
@@ -11,6 +11,12 @@ const config = require("./config");
...
+if ([origin, AppID, APP_SECRET, redirect_url, cookieAge].includes("")) {
+  console.warn("[WARNING] config.js has at least one empty property.");
+}
+}
```
Comment (Steven-Chan) [inline src/app.js:19]: Code duplicated.

**Review by Steven-Chan — CHANGES_REQUESTED** [review]

File: src/app.js (hunk 2 above)
Comment (CMDRyan) [reply src/app.js]: I believe it was caused by just committing the added lines while forgetting to commit the removed ones as well. 

@Steven-Chan 
lines removed

**Pull request merged**